tests/coins/btc/test_lightning.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from bitcart.errors import LightningDisabledError
+
+pytestmark = pytest.mark.asyncio
+
+
+async def test_node_id(btc_wallet):
+    assert (await btc_wallet.node_id, str)
+
+
+async def test_add_invoice(btc_wallet):
+    assert await btc_wallet.addinvoice(0.5, "test description")
+
+
+async def test_lightning_disabled(btc_wallet):
+    await btc_wallet.set_config("lightning", False)
+    with pytest.raises(LightningDisabledError):
+        await btc_wallet.node_id
```

Comment:
I think after this test lightning should be back on. Or even better, it could be turned off during the test and on after the test via `pytest.yield_fixture`